.travis.yml

```diff
@@ -74,6 +74,7 @@ install:
   - HADDOCK=${HADDOCK-true}
   - INSTALLED=${INSTALLED-true}
   - GHCHEAD=${GHCHEAD-false}
+  - CABALFLAGS="--flags='quickcheck-classes -aeson -semigroupoids'"
```

Comment:
I think I goofed—it turns out `cabal-install` doesn't like it when you surround this with double-quotes. I think it should work if you remove them.